src/backend/Csrs.Api/Authentication/AuthenticationExtensions.cs

```diff
@@ -2,67 +2,66 @@
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using System.Configuration;
 
-namespace Csrs.Api.Authentication
+namespace Microsoft.Extensions.DependencyInjection;
```

Comment:
Changed namespace so that in Program.cs, we do not require additional using statements and the `AddXXX` method is just available.